pkg/pod/status_test.go

```diff
@@ -238,10 +238,11 @@ func TestSetTaskRunStatusBasedOnStepStatus_sidecar_logs(t *testing.T) {
 			ctx := config.ToContext(context.Background(), &config.Config{
 				FeatureFlags: featureFlags,
 			})
-			var wantErr *multierror.Error
-			wantErr = multierror.Append(wantErr, c.wantErr)
+			wantErr := errors.Join(c.wantErr)
```

Comment:
This shouldn't be necessary anymore, since we're no longer dealing the with `MultiError` custom type, and just dealing with the `error` type. We should be able to get away with just using `c.wantErr` without a wrapper. See [this example](https://go.dev/play/p/VI4AXVGpjoQ)